2018-07-21 13:02:58 INFO  EmployeeHiberDao:32 - Exiting from saveEmployee :: EmployeeDao
2018-07-21 13:03:21 INFO  EmployeeController:39 - Entered into saveEmployee :: EmployeeController
2018-07-21 13:03:21 INFO  EmployeeController:40 - -----------------------------------------------------------
2018-07-21 13:03:21 INFO  EmployeeController:54 - suchi...null...CSE...102---4532---Hyderabad
2018-07-21 13:03:21 INFO  EmployeeController:57 - suchi...null...CSE...102---4532---Hyderabad
2018-07-21 13:03:21 INFO  EmployeeController:60 - #####################################33
2018-07-21 13:03:21 DEBUG EmployeeController:61 - 
2018-07-21 13:03:21 DEBUG EmployeeController:62 - 
2018-07-21 13:03:21 INFO  EmployeeController:63 - #####################################33
2018-07-21 13:03:21 INFO  EmployeeHiberDao:20 - Entered saveEmployee :: EmployeeDao
2018-07-21 13:03:22 INFO  EmployeeHiberDao:25 - trying to save!!
2018-07-21 13:03:22 INFO  EmployeeHiberDao:28 - Done!!
2018-07-21 13:03:22 INFO  EmployeeHiberDao:32 - Exiting from saveEmployee :: EmployeeDao
2018-07-26 20:35:00 INFO  EmployeeController:39 - Entered into saveEmployee :: EmployeeController
2018-07-26 20:35:00 INFO  EmployeeController:40 - -----------------------------------------------------------
2018-07-26 20:35:00 INFO  EmployeeController:54 - suchi...null...CSE...102---4532---Hyderabad
2018-07-26 20:35:00 INFO  EmployeeController:57 - suchi...null...CSE...102---4532---Hyderabad
2018-07-26 20:35:00 INFO  EmployeeController:60 - #####################################33
2018-07-26 20:35:00 DEBUG EmployeeController:61 - 
2018-07-26 20:35:00 DEBUG EmployeeController:62 - 
2018-07-26 20:35:00 INFO  EmployeeController:63 - #####################################33
2018-07-26 20:35:00 INFO  EmployeeHiberDao:20 - Entered saveEmployee :: EmployeeDao
2018-07-26 20:35:10 INFO  EmployeeHiberDao:25 - trying to save!!
2018-07-26 20:35:10 ERROR EmployeeHiberDao:30 - Exception occured while saving an employee
org.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:268)
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:184)
	at org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)
	at org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)
	at org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1216)
	at org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:383)
	at org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:133)
	at com.facebook.daos.EmployeeHiberDao.saveEmployee(EmployeeHiberDao.java:27)
	at com.facebook.controllers.EmployeeController.saveEmployee(EmployeeController.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.bind.annotation.support.HandlerMethodInvoker.invokeHandlerMethod(HandlerMethodInvoker.java:176)
	at org.springframework.web.servlet.mvc.annotation.AnnotationMethodHandlerAdapter.invokeHandlerMethod(AnnotationMethodHandlerAdapter.java:440)
	at org.springframework.web.servlet.mvc.annotation.AnnotationMethodHandlerAdapter.handle(AnnotationMethodHandlerAdapter.java:428)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:919)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:851)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:953)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:844)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:635)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:829)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:742)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:496)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:650)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:803)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:790)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1468)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.sql.BatchUpdateException: Duplicate entry '' for key 'PRIMARY'
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1666)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1082)
	at org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)
	at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)
	... 45 more
